rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -73,6 +78,47 @@ def generate_py(generator_arguments_file, typesupport_impls):
         idl_file = parse_idl_file(locator)
         idl_content.elements += idl_file.content.elements
 
+    # NOTE(sam): remove when a language specific name mangling is implemented
+
+    def print_warning_if_reserved_keyword(member_name, interface_type, interface_name):
+        if (keyword.iskeyword(member.name)):
+            print(
+                "Member name '{}' in the {} '{}' is a "
```

Comment:
is it really not supported? or you just have to use everywhere:
```python3
getattr(msg, 'name_that_is_also_a_keyword')
```
which is ugly, but supported.

Reply:
Are you suggesting we just say "is not recommended", rather than "not supported"?

Reply:
Based on the below [answer](https://github.com/ros2/rosidl_python/pull/96#issuecomment-582893866), I think that not supported is the correct wording.

----

If we want to support message fields named as keywords in `Python`, an alternative to do name mangling is to use `getattr` everywhere in the generated files. The user would have to use the same in their programs (in case the field name was a keyword). Of course, this alternative is not viable in other programming languages (e.g.: c++), but for `Python` I actually prefer that than mangling (though it's arguable).